src/instructlab/sdg/generate_data.py

```diff
@@ -115,20 +125,21 @@ def _gen_train_data(
 
 def _knowledge_seed_example_to_test_data(seed_example, system_prompt):
     res = []
-    for qna in seed_example["questions_and_answers"]:
-        user = qna["question"] + "\n" + seed_example["context"]
-        res.append(
-            {
-                "system": system_prompt,
-                "user": _unescape(user),
-                "assistant": _unescape(qna["answer"]),
-            }
-        )
+    for i in range(3):
```

Comment:
Should we also have this either documented or parameterize the number 3 as to why we are sticking to this value?
```suggestion
    for i in range(num_iterations):
```